packages/core/lib/commands/test/run.js

```diff
@@ -30,6 +32,25 @@ const parseCommandLineFlags = options => {
   }
 };
 
+// Sanitize ganache options specified in the config
```

Comment:
Small nit. I think this comment is unnecessary because intent is already clearly communicated by the names you chose.

Reply:
Thanks @cds-amal for the suggestion. I will remove this in my upcoming PR.